src/sinks/influxdb/mod.rs

```diff
@@ -174,6 +189,7 @@ fn healthcheck(
 
 // https://v2.docs.influxdata.com/v2.0/reference/syntax/line-protocol/
 fn influx_line_protocol(
+    protocol_version: ProtocolVersion,
```

Comment:
What about pass `protocol_version` as a reference?

```suggestion
    protocol_version: &ProtocolVersion,
```

Reply:
Reference should not give any advantages for such simple enum